### DISCUSSION TITLE
App.vue

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#517] [Q&A] [answered]
Buenas noches cuando abro el app.vue, me sale esto y solo una vez que lo cierro desaparece. ¿Alguien sabe como arreglarlo o asi se deja?

![imagen](https://user-images.githubusercontent.com/75102777/160052995-7315e300-99e1-49f8-8113-6fed8ca84f80.png)

**Comment by miguelapol** [comment, marked as answer]
Creo que es ponerlo dentro de un div🤔

**Reply by MauroMontan** [reply]
si, el nav y el router-view tienen que estar en un mismo div.Si se van a colocar más de 2 elementos dentro del template, tiene que haber un componente padre

**Reply by GloriaCoralCerecedo** [reply]
Oh vale puse el router -view dentro del nav y funciono, gracias!
